vars/pipelineVars.groovy

```diff
@@ -31,6 +30,10 @@ class pipelineVars implements Serializable {
     String prodCluster = "api.insights.openshift.com"
     String devCluster = "api.insights-dev.openshift.com"
 
+    String jenkinsSlaveImage = 'openshift/jenkins-slave-base-centos7:v3.11'
```

Comment:
In the OSD cluster I only see `openshift/jenkins-slave-base-centos7:latest` -- there's no imagestream tag for v3.11 -- do you think we can use `latest` in both clusters?

Reply:
it's from here https://hub.docker.com/r/openshift/jenkins-slave-base-centos7/tags